src/helpers/tracking.js

```diff
@@ -0,0 +1,65 @@
+/**
+ */
+export default function initializeCookieConsentBar() {
+    window.addEventListener("load", () => {
+        var shouldAddMotionListeners = true;
+
+        window.cookieconsent.initialise({
+            position: "bottom",
+            content: {
+                message: "We use cookies when you continue to use our website.",
+                dismiss: "got it!",
+                link: "Learn more",
+                href: "https://www.bunq.com/privacy"
+            },
+            container: document.querySelector(".react-root"),
+            onInitialise: function () {
+                if (this.hasConsented()) {
+                    initializeAllTracking();
+
+                    shouldAddMotionListeners = false;
+                }
+            }
+        });
+
+        if (window.cookieconsent.hasInitialised && shouldAddMotionListeners) {
+            window.addEventListener("scroll", initializeAllTracking);
+            window.addEventListener("keydown", initializeAllTracking);
+            window.addEventListener("mousedown", initializeAllTracking);
+            window.addEventListener("touchstart", initializeAllTracking);
+        }
+    });
+}
+
+/**
+ */
+function initializeGoogleTagManagerIfNeeded() {
```

Comment:
maybe change the order of this and the below fn, it's confusing like this

Reply:
done